docs/UserGuide.md

```diff
@@ -43,144 +42,260 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
+* `list work`<br>
+  List all unfinished tasks in list `work`
+* `list study favourite`<br>
+  List all favourite tasks in list `study`
+* `list lists`<br>
+  List all listname with list indices
+
+### 2.4. Adding a new list : `add list`
+
+Add a new list in Dueue<br>
+Format: `add list LISTNAME`
+
+> * Listname cannot be the same as other existing lists
+> * Listname cannot begin with numbers
+
+Examples:
+
+* `add list school`
+
+### 2.5. Updating a list : `update list`
+
+Update a list in Dueue with new name<br>
+Format: `update list LISTNAME NEW_LISTNAME`
+Format: `update list LISTINDEX NEW_LISTNAME`
+
+> * Listname cannot be the same as other existing lists
+> * Listname cannot begin with digits
+> * Listname cannot contain whitespaces
+> * Do not need to specify whether the parameter is LISTINDEX or LISTNAME
+
+Examples:
+
+* `update list school newSchool`<br>
+  Update the name of list `school` to `newSchool`
+* `update list oldList newList`<br>
+  Update the name of list `oldList` to `newList`
+
+### 2.6. Updating task(s) : `update`
 
-Edits an existing person in the address book.<br>
-Format: `edit INDEX [NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Updates existing task(s) in Dueue<br>
+Format: `update TASKINDEX ... [n/NAME] [due/DUEDATE] [t/TIME] [#LISTNAME] [d/DESCRIPTION] [@VENUE] [p/PRIORITYLEVEL] [FAVOURITE]`
 
-> * Edits the person at the specified `INDEX`.
-    The index refers to the index number shown in the last person listing.<br>
-    The index **must be a positive integer** 1, 2, 3, ...
+> * Edits the task at the specified `INDEX`.
+> * The index refers to the index number shown in the last listing.<br>
+> * The index **must be a positive integer** 1, 2, 3, ...
 > * At least one of the optional fields must be provided.
 > * Existing values will be updated to the input values.
-> * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-> * You can remove all the person's tags by typing `t/` without specifying any tags after it. 
+> * You can remove details by typing nothing after the `/` or `@`without specifying any tags after it eg,`due/`.
+> * To update multiple entries, indices must be seperated by whitespaces.
+> * Specifications of fields for task can be entered in any order.
 
 Examples:
 
-* `edit 1 p/91234567 e/johndoe@yahoo.com`<br>
-  Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@yahoo.com` respectively.
+* `update 1 Laundry due/every sunday @`<br>
+  Updates the name, due date and the venue of the 1st task to be `Laundry`, `every sunday` and remove the venue respectively.
 
-* `edit 2 Betsy Crower t/`<br>
-  Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `update 2 due/ &f`<br>
+  Remove the due date of the task with `INDEX` 2 and make it not favourite.
 
-### 2.5. Finding all persons containing any keyword in their name: `find`
+### 2.7. View certain tasks: `view`
```

Comment:
2.7 view

- Should this command be merged with or better distinguished from the "list"? 

- Similar date specifier issue, usage of favourite.

- Typo in "view all tasks", the Format should be "view all"?